### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by estebanmual**
In this milestone I: 
  - Added React Redux
  - Structured the application files using a "feature folder" approach
  - Wrote the book's actions and reducer using the ducks pattern.
    - Defined action types for adding and removing a book.
    - Set the initial state to be an empty array of books.
    - Exported Action Creators.
    - Wrote the reducer and exported it as default.
      - Defined state changes for the actions
      - In case of unknown action - return the current state.
  - Write your categories actions and reducer using the ducks pattern.
    - Defined an action type for checking the status.
    - Set the initial state to be an empty array of categories.
    - Exported Action Creators.
    - Wrote the reducer and exported it as default.
      - For the check status action return a string "Under construction".
      - In case of unknown action - return the current state.
  - Configure the Redux Store
    - Combine both reducers into a root reducer by using `combineReducers` function. 
    - Use the root reducer to create your store.

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { combineReducers, createStore } from 'redux';
+import booksReducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+const rootReducer = combineReducers({ booksReducer, categoriesReducer });
```
Comment (GzimAsani) [inline src/redux/configureStore.js]: - [x] Please give a key to your reducers for example
- books: bookReducer
- status: statusReducer

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi  @estebanmual       
Good job so far! You are almost done :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good job with the git-flow :+1:
- Good PR description :clap: 

## Required Changes ♻️ 
- [x] _Check the comments below :arrow_double_down: _

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/configureStore.js (hunk 1 above)
Comment (estebanmual) [reply src/redux/configureStore.js]: Thank you so much for your feedback @GzimAsani! I will implement this change as you have asked.  😎 👍

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @estebanmual , 

Great work on making the changes requested by the previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**